book/src/drivers.md

```diff
@@ -66,6 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
+[KaiXiangTech MegaBric](https://github.com/kaixiangtech/csi-driver-flexblock) | `flexblock.csi.kaixiangtech.com` | v1.6.0 | A Container Storage Interface (CSI) plugin for KaiXiangTech MegaBric Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion, Cloning
```

Comment:
The version column refers to the CSI spec version, not the CSI driver version. The latest CSI spec version is 1.5: https://github.com/container-storage-interface/spec/releases/tag/v1.5.0

Reply:
we test success in new version
change support version v1.5.0